assets/newspack-ui/scss/variables/_colors.scss

```diff
@@ -20,6 +20,10 @@
 	--newspack-ui-color-alert-green-bg: #edfaef;
 	--newpsack-ui-color-alert-red-bg: #fcf0f1;
 
+	// Theme variables - from the classic theme; overridden in class-newspack-ui.php for block theme.
+	--newspack-ui-color-primary: var(--newspack-theme-color-primary);
+	--newspack-ui-color-against-primary: var(--newspack-theme-color-against-primary);
```

Comment:
NIT

```suggestion
	--newspack-ui-color-primary: var( --newspack-theme-color-primary );
	--newspack-ui-color-against-primary: var( --newspack-theme-color-against-primary );
```

Reply:
Good catch -- thanks!! 🤦‍♀️ Fixed in [d2a61cf](https://github.com/Automattic/newspack-plugin/pull/2877/commits/d2a61cf99d642c06727dd0b1aad20b1a4d8a8a76).